.gitpod.yml

```diff
@@ -41,6 +46,7 @@ tasks:
       MM_SERVICESETTINGS_ENABLEOAUTHSERVICEPROVIDER: true
       MM_SERVICESETTINGS_ENABLEDEVELOPER: true
       MM_SERVICESETTINGS_ENABLETESTING: true
+      MM_TEAMSETTINGS_ENABLEOPENSERVER: true
```

Comment:
I find it strange that we have to do this. The issue might might be associated with just a subset of tests that require this, which may explain why I haven't run into this.

Adding more variables to this is making me think that we should have these environment variables in `.gitpod.Dockerfile`, to ensure all terminals created by the user will also have these set. I can do this after this PR is merged

Reply:
@mickmister That makes sense; thanks!